client/src/assets/color-palette.svg

```diff
@@ -0,0 +1 @@
5.725zm45.725-30.725c16.941 0 30.725 13.783 30.725 30.725s-13.783 30.725-30.725 30.725-30.725-13.783-30.725-30.725 13.783-30.725 30.725-30.725z"/><path d="m46.142 299.061c0 113.139 97.467 199.138 211.203 206.066.155.009.31.014.463.014 3.934 0 7.237-3.065 7.479-7.044.252-4.134-2.896-7.69-7.03-7.942-110.532-6.733-197.115-90.671-197.115-191.094 0-113.779 108.333-201.572 232.132-190.358 5.632 5.869 12.562 10.573 19.814 14.016-.595 2.741-.491 5.665.229 8.308l14.844 54.597c-3.456 4.039-4.657 9.538-3.278 14.601l3.533 12.998c2.149 7.917 10.06 12.981 18.146 11.579l7.006 30.339c-25.334 11.541-39.187 26.996-39.187 43.921 0 16.985 14.158 32.697 39.866 44.239 9.142 4.104 17.053 9.729 23.542 16.732 8.25 36.216 10.787 47.707 16.834 65.913-5.598 24.418-22.336 43.136-46.187 51.535-18.198 6.408-37.138 10.513-56.293 12.201-4.126.364-7.177 4.003-6.813 8.129.363 4.127 4.016 7.172 8.129 6.813 20.404-1.798 40.578-6.17 59.959-12.995 22.063-7.769 39.483-23.509 49.307-43.805 10.03 24.608 19.172 39.815 27.671 54.636 4.296 7.495 13.12 11.147 21.459 8.88 8.337-2.267 14.097-9.883 14.009-18.523-.258-25.124.404-62.891-17.248-119.162-1.24-3.952-5.447-6.151-9.401-4.911-3.952 1.24-6.15 5.449-4.911 9.401 16.57 52.816 16.284 87.857 16.561 114.826.02 1.816-1.191 3.417-2.944 3.894-1.75.477-3.608-.291-4.512-1.867-31.556-55.031-35.532-71.354-51.44-140.232 0 0-30.904-133.833-30.904-133.834l22.039-5.992 11.17 35.604v.001l29.55 94.192c1.24 3.952 5.446 6.151 9.401 4.911 3.952-1.24 6.151-5.449 4.911-9.401l-27.351-87.183c41.323-16.743 52.882-72.241 17.505-103.134-14.497-12.659-30.748-23.457-48.312-32.184l-1.227-4.512c-.749-2.758-2.134-5.231-4.019-7.272 7.571-11.179 15.125-28.469 9.624-48.703-5.281-19.421-18.264-26.914-32.008-34.846-8.336-4.811-16.956-9.786-25.272-18.063-8.081-8.039-21.429-4.428-24.33 6.614-2.98 11.349-7.896 20.004-12.648 28.374-7.923 13.953-17.572 30.191-7.352 53.648-127.314-5.481-236.604 86.862-236.604 206.075zm326.105 36.976c-3.739-2.397-7.688-4.549-11.856-6.42-19.708-8.849-31.011-19.986-31.011-30.556 0-10.062 9.958-20.399 27.638-28.975zm46.175-174.807c26.551 23.185 17.988 64.539-12.139 77.489-7.747-24.692-5.646-17.997-8.717-27.786 7.845-3.003 11.893-11.41 9.784-19.172l-3.533-12.998c-1.385-5.089-5.228-9.21-10.221-10.931l-8.212-30.203c11.824 6.837 22.889 14.738 33.038 23.601zm-30.315 20.761.003-.001c.482-.13 1.076.134 1.231.708l3.533 12.997c.146.536-.172 1.09-.707 1.235-10.036 2.729-38.893 10.574-48.043 13.062-.524.146-1.086-.162-1.235-.707l-3.533-12.998c-.155-.573.238-1.108.707-1.235zm-9.715-12.903-36.485 9.919-14.115-51.917c-.365-1.335.472-2.726 1.762-3.078l31.646-8.604c1.229-.337 2.689.334 3.077 1.762 4.642 17.075 1.386 5.099 14.115 51.918zm-75.25-122.341c4.882-8.596 10.4-18.315 13.913-31.21 9.524 9.325 19.222 14.921 27.796 19.87 12.408 7.161 21.373 12.334 25.031 25.79 4.471 16.442-3.226 30.778-9.602 39.195-.966.087-2.097.293-3.017.543l-31.646 8.604c-.965.262-2.071.679-2.873 1.058-13.267-5.445-24.335-14.98-28.129-28.936-3.658-13.456 1.453-22.457 8.527-34.914z"/></g></svg>
```

Comment:
There are some not used properties in your SVGs. These properties are created by design tools like Adobe XD.
For example, you can remove id and data-name.
Take a look at https://www.svgminify.com/

Similar to Hanttone/capstone-project#8 (comment)

Reply:
thanks for the great tip!